locales/en-US/pocket.properties

```diff
@@ -26,6 +26,8 @@ tagline = Save articles and videos from Firefox to view in Pocket on any device,
 taglinestory_one = Click the Pocket Button to save any article, video or page from Firefox.
 taglinestory_two = View in Pocket on any device, any time.
 tagssaved = Tags Added
+tos = By continuing, you agree to Pocket's *Terms of Service* and *Privacy Policy*
```

Comment:
Thanks. If we can agree on the placeholders, I can send the email later today and merge this PR.